paddle/fluid/pybind/eager_utils.cc

```diff
@@ -34,6 +34,7 @@ limitations under the License. */
 namespace paddle {
 namespace pybind {
 
+using PyNameVarBaseMap = std::unordered_map<std::string, py::handle>;
```

Comment:
bad name, it seems not VarBaseMap

Reply:
This name is not used, will be removed. nkjkjkj